tasks/test-tutorial

```diff
@@ -34,7 +34,7 @@ if [ -z "$1" ]
   then
   cd "$SCRIPTPATH/../"
   # build all the packages
-  yarn build:clean && yarn lerna run build:js
+  yarn build:clean && yarn build
```

Comment:
@peterp When I was testing https://github.com/redwoodjs/redwood/pull/1824 via E2E script and then follow-up manual checks, I released we weren't building ts during the process. Other than it taking longer, is there a reason _not_ to build ts? 

I'm not sure we need it for the E2E itself, but rather for possible tests/checks added in the future that we perform after the E2E. 

Will defer to you. Thanks!

Reply:
The only reason why I'm not doing it is because it takes longer, in order for anything to be merged into main we already need to have the types pass, so I figured it's ok to skip it.